site/docs/reference/Connectors/capture-connectors/netsuite.md

```diff
@@ -0,0 +1,68 @@
+---
+sidebar_position: 1
+---
+# NetSuite
+
+This connector captures data from NetSuite into Flow collections.
+
+It is available for use in the Flow web application. For local development or open-source workflows, [`ghcr.io/estuary/source-netsuite:dev`](https://ghcr.io/estuary/source-netsuite:dev) provides the latest version of the connector as a Docker image. You can also follow the link in your browser to see past image versions.
+
+This connector is based on an open-source connector from a third party, with modifications for performance in the Flow system.
+You can find their documentation [here](https://docs.airbyte.com/integrations/sources/netsuite/),
+but keep in mind that the two versions may be significantly different.
+
+## Supported data resources
+
+- Streams are generated based on `ROLE` and `USER` access to them as well as `Account` settings, make sure you're using the correct role assigned in our case `airbyte-integration-role` or any other custom `ROLE` granted to the Access Token, having the access to the NetSuite objects for data sync.
```

Comment:
"streams" is airbyte verbiage we want to avoid (except in the resource config, where we have less control) - we say "resources." Also, `airbyte-integrations-role` should probably be changed. I'm not actually very clear based on reading this what the role does, ad if there's more prerequisites that might need to be done to get this to actually work?

Reply:
I just took a look at the Airbyte docs for this connector. It looks like you create a custom role as part of the setup (in their case, `airbyte-integrations-role`, so we probably wanna tell people to create `estuary-role`). And any objects to which you grant that role access are captured. 

So two action items here:
* Revise this section to say something like "Flow captures collections from any NetSuite object to which you grant access during setup[Link to Setup section], including `Transactions`, `Reports`, `Lists`, and `Setup`."  Don't worry about linking Netsuite docs to object types, as the NetSuite docs are incredibly confusing. 
* Expand the prerequisites section and add a Setup section based off Airbyte's prerequisites. Something worth noting: Airbyte talks about prerequisites as what's needed _before_  you begin setup; whereas we talk about the prerequisites in terms of what needs to be ready immediately before using Flow. So the Setup steps help users _meet_ the prerequisites. Hope that makes sense -- for an example see https://docs.estuary.dev/reference/Connectors/capture-connectors/PostgreSQL/#prerequisites. For Netsuite, looks like the actual prereqs are: new integration with token-based auth; a custom role with access to objects you want to capture; a new user assigned that role; access token generated for the role.

Reply:
Hi @oliviamiannone I've made those changes, let me know if they look fine to you, thanks!

Reply:
Hey Jenny -- This looks fine for the time being, considering where we're at with the connector! In the future we can add more details / links.

Also, want to clarify my comment about the Netsuite docs -- I don't think they're helpful to link for the _suported resources_ but I do think they'd be helpful to link for some of the prerequisites, since we don't want to write out a detailed step by step like airbyte. EG, the docs for a new integration are https://docs.oracle.com/en/cloud/saas/netsuite/ns-online-help/section_157771733782.html But again it's fine to hold off and just get this merged.